index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
**Project Requirement** 
> - Add a `.html` file that includes a header with the text "Hello Microverse!". Use h1 tag for that.  

🤓 According to project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you should add your `h1` element inside the `header` semantic tag. I suggest always using semantic tags 🏷️ in your HTML projects as semantics tags add accessibility 💯  to your app 😄 .